test/functorch/test_control_flow.py

```diff
@@ -1664,6 +1667,75 @@ def forward(self, arg0_1, arg1_1, arg2_1):
     """,  # noqa: B950
         )
 
+    @skipIfTorchDynamo("Graph is not captured by backend if test with dynamo")
+    @requiresCrossRef
+    def test_cond_simple_with_linear_compile_check_graph_crossref(self):
```

Comment:
What are these tests testing? That cond, while_loop work with CrossRefMode ?

Reply:
Yeah, so the issue here is that now that we trace TorchFunctionModes with dynamo when we run these tests under dynamo, with CrossRefMode the arg order changes so we need different ref strings. So, I separated out the test into two versions, one w/ cross ref and one without

Reply:
Hmm, how difficult is it to make the CrossRef tests have the same argument order? It's going to be difficult to future developers to update these tests because they won't know to run the tests under CrossRefMode until the tests start failing in CI.

Reply:
I don't think this is solvable unfortunately, basically, the order in which tensors are accessed in the code determine this. Since every mode is different (and absence of mode) args will not be guaranteed to be in the same order, and there is no way of knowing the "correct" order.

Reply:
Ah I see. In that case, I'd prefer to delete the crossref-specific tests

Reply:
I'll remove